server/models.py

```diff
@@ -0,0 +1,72 @@
+from app import db
+from sqlalchemy.sql import func
+from sqlalchemy import ForeignKey
+
+class contest_user(db.Model):
```

Comment:
for classes, we should be naming them in PascalCase

```suggestion
class ContestUser(db.Model):
```

Reply:
all Users should be the same so this may not be required